### PR TITLE
property duplicate

### PR DESCRIPTION
**Pull request by yasintz**
https://github.com/nicolasdao/graphql-s2s/issues/33

### CODE REVIEW

File: src/graphqls2s.js [hunk 1]
```diff
@@ -806,7 +806,13 @@ let graphqls2s = {
 		.val(),
 	transpileSchema: graphQlSchema => chain(resetMemory())
 		.next(() => removeGraphMetadata(graphQlSchema))
-		.next(data => buildSchemaString(getSchemaParts(data.stdSchema, data.metadata)))
+		.next(data =>
+			buildSchemaString(
+				_.map(getSchemaParts(data.stdSchema, data.metadata), item => (Object.assign(
+					item,{blockProps: _.uniqBy(item.blockProps, 'value')}
```
Comment (yasintz) [inline src/graphqls2s.js]: may be better to perform in upper functions

Comment (yasintz) [reply src/graphqls2s.js]: can you do it ?

**Comment by yasintz** [comment]
@nicolasdao

**Comment by nicolasdao** [comment]
That's great @yasintz!

I've just reviewed your latest changes and your unit tests (26 and 27). That's crystal clear! I've just published your changes to npm. The latest version (0.19.2) contains your fixes. 

On behalf of our community, thanks a lot. :+1: :sparkles: :rocket:

Nic

**Pull request merged**